support/nginx/peertube

```diff
@@ -1,27 +1,49 @@
 server {
   listen 80;
-  # listen [::]:80;
-  server_name domain.tld;
+  listen [::]:80;
+  server_name peertube.example.com;
 
-  access_log /var/log/nginx/peertube_access.log;
-  error_log /var/log/nginx/peertube_error.log;
+  access_log /var/log/nginx/peertube.example.com.access.log;
+  error_log /var/log/nginx/peertube.example.com.error.log;
 
-  location /.well-known/acme-challenge/ { allow all; }
-  location / { return 301 https://$host$request_uri; }
+  rewrite ^ https://$server_name$request_uri? permanent;
 }
 
 server {
   listen 443 ssl http2;
-  # listen [::]:443 ssl http2;
-  server_name domain.tld;
-
-  access_log /var/log/nginx/peertube_access.log;
-  error_log /var/log/nginx/peertube_error.log;
-
-  # For example with Let's Encrypt
-  ssl_certificate      /etc/letsencrypt/live/domain.tld/fullchain.pem;
-  ssl_certificate_key  /etc/letsencrypt/live/domain.tld/privkey.pem;
-  ssl_trusted_certificate /etc/letsencrypt/live/domain.tld/chain.pem;
+  listen [::]:443 ssl http2;
+  server_name peertube.example.com;
+
+  # For example with Let's Encrypt (you need a certificate to run https)
+  ssl_certificate      /etc/letsencrypt/live/peertube.example.com/fullchain.pem;
+  ssl_certificate_key  /etc/letsencrypt/live/peertube.example.com/privkey.pem;
+  ssl_dhparam /etc/nginx/dhparam.pem; # openssl dhparam -out /etc/nginx/dhparam.pem 4096
```

Comment:
I think we'll need to put `openssl dhparam -out /etc/nginx/dhparam.pem 4096` in the installation guide

Reply:
I think DH for key exchange is already depreciated (only ECDH is the correct way nowdays), so no need for dhparams anyway.